templates/layouts/petition-and-scroll-to-share.liquid

```diff
@@ -0,0 +1,130 @@
+  </div>
+</div>
+
+<div class="center-content center-content--accomodates-stuck-footer petition-and-scroll-to-share__yes-no-question-wrapper">
+  <div class="center-content__one-column">
+
+    <div class="center-content__central-square">
+
+      {% capture message %}{{ 'petition.thank_you' | val: 'petition_title', title | t }}{% endcapture %}
+      <h1 class="thank-you__thanks">{{ message }}</h1>
+      <div class="thank-you__cta">
+        <span class="two-step__question">{{ 'share.two_step.cta' | t }}</span>
+        <span class="two-step__declined hidden-closed">{{ 'share.two_step.declined' | t }}</span>
+        <span class="two-step__accepted hidden-closed">{{ 'share.two_step.accepted' | t }}</span>
+      </div>
+
+      <div class="center-content__centered-element">
+        <div class="two-step__question">
+          <div class="share-buttons">
+            <div class="share-buttons__button button two-step__button two-step__accept">
+              {{ 'share.two_step.accept' | t }}
+            </div>
+            <div class="share-buttons__button button two-step__button two-step__decline">
+              {{ 'share.two_step.decline' | t }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+  </div>
+</div>
+
+<div class="center-content center-content--accomodates-stuck-footer petition-and-scroll-to-share__share-wrapper">
+  <div class="center-content__one-column">
+
+    <div class="center-content__central-square">
+
+      <div class="thank-you__cta">{{ 'share.cta' | t }}</div>
+
+      <div class="center-content__centered-element">
+        {% include 'Share' %}
+      </div>
+    </div>
+
+  </div>
+</div>
+
+<div class="center-content petition-and-scroll-to-share__fundraiser-wrapper">
+  <h1 class="thank-you__cta intro"> {{ 'petition_and_scroll.fundraiser_intro' | t }} </h1>
+  {% capture fundraiser_title %} {{ 'share_and_donate.fundraiser_title' | t }} {% endcapture %}
+  {% include 'Fundraiser Refactor', freestanding: true, fundraiser_title: fundraiser_title %}
+</div>
+
+{% include 'Petition Mobile Footer' %}
+{% include 'Small Image Footer' %}
+
+<script type="text/javascript">
+  $(document).ready(function(){
+    var petitionOverlayButton = $('.petition-bar__mobile_ui__bottom_bar');
+    var petitionSidebar = $('.center-content__fixed-right');
+    var yesNoQuestion = $('.petition-and-scroll-to-share__yes-no-question-wrapper');
+    var shareStep = $('.petition-and-scroll-to-share__share-wrapper');
+    var fundraiserStep = $('.petition-and-scroll-to-share__fundraiser-wrapper');
+
+    new window.champaign.Petition({
+      redirectAfterAction: false,
+      submissionCallback: petitionCallback
+    });
+
+    $('.two-step__accept').click(displayAndScrollToShare);
+    $('.two-step__decline').click(displayAndScrollToDonate);
+
+    function makeStepFullScreen(stepElement) {
+      var offset = 35;
+      var padding = parseInt(stepElement.css('padding-top'));
```

Comment:
developer.mozilla say you should always pass the radix - to avoid confusion and for consistent behaviour across browsers.